railgun/internal/ctrlutils/utils.go

```diff
@@ -135,3 +138,24 @@ func IsIngressClassSpecConfigured(obj client.Object, expectedIngressClassName st
 	}
 	return false
 }
+
+// GetSemver attempts to provide a semver.Version for the provided semver string and produces an error if it can not be parsed.
+func GetSemver(v string) (semver.Version, error) {
```

Comment:
This whole function can be removed now that kong.ParseSemanticVersion is used, I guess?

Reply:
Yep! thanks for catching that for me, I had forgot: f349a8ee3144099ab93881e57247ca14db368279 :+1: